<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerClass</class>
 <widget class="QMainWindow" name="ImageViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>689</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1632</width>
     <height>636</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1632</width>
     <height>636</height>
    </size>
   </property>
   <widget class="QGraphicsView" name="referenceView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QGraphicsView" name="floatView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QSlider" name="referenceSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="floatSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="referenceSliderLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Float Image</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="referenceSpinBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="floatSliderLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="floatSpinBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>550</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="referenceShowMask">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Mask</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="floatShowMask">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Mask</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>1031</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Shared Control</string>
    </property>
    <widget class="QSlider" name="windowSlider">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="referenceSliderLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Window</string>
     </property>
    </widget>
    <widget class="QLabel" name="sharedSliderName">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Depth</string>
     </property>
    </widget>
    <widget class="QSlider" name="sharedSlider">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="sharedSpinBox">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="sharedShowMask">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mask</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="subtractView">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="subtractSliderLabel">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>550</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QSlider" name="subtractSlider">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>550</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="subtractSpinBox">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>550</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtract Image</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>referenceView</zorder>
   <zorder>floatView</zorder>
   <zorder>referenceSlider</zorder>
   <zorder>floatSlider</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>referenceSliderLabel</zorder>
   <zorder>label_5</zorder>
   <zorder>referenceSpinBox</zorder>
   <zorder>floatSliderLabel</zorder>
   <zorder>floatSpinBox</zorder>
   <zorder>referenceShowMask</zorder>
   <zorder>floatShowMask</zorder>
   <zorder>subtractView</zorder>
   <zorder>subtractSliderLabel</zorder>
   <zorder>subtractSlider</zorder>
   <zorder>subtractSpinBox</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageViewer.qrc"/>
 </resources>
 <connections/>
</ui>
